AsyncDisplayKit/ASTextNode.mm

```diff
@@ -394,26 +400,23 @@ - (void)setAttributedText:(NSAttributedString *)attributedText
 
   // Tell the display node superclasses that the cached layout is incorrect now
   [self invalidateCalculatedLayout];
-
```

Comment:
I wonder if we need to include the renderer invalidation and the setNeedsDisplay in the lock context?  If we don't, what happens if another thread writes to other properties in the meantime?